app/views/subscriptions/index.html.erb

```diff
@@ -38,20 +38,24 @@
 <dl class="subscription index group">
   <% @subscriptions.each do |subscription| %>
     <dt>
-      <%= link_to(subscription.name, subscription.subscribable) %>
+      <% if subscription.subscribable.nil? %>
+      <%= t(".deleted") %>
```

Comment:
Can you indent this two spaces? And can we use `<%= subscription.name %>` here instead now that deleted subscriptions have a `name`? That would save Translation from having to translate the same thing twice.